tests/cypress/e2e/form/destination_config_fields/request_source.cy.js

```diff
@@ -0,0 +1,90 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+describe('Request source configuration', () => {
+    beforeEach(() => {
+        cy.login();
+        cy.changeProfile('Super-Admin', true);
+
+        // Create form
+        cy.createFormWithAPI().visitFormTab('Form');
+
+        // Add a default question
+        cy.findByRole('button', { 'name': "Add a new question" }).click();
+        cy.findByRole('button', { 'name': 'Save' }).click();
+
+        // Go to destination tab
+        cy.findByRole('tab', { 'name': "Items to create" }).click();
+        cy.findByRole('button', { 'name': "Add ticket" }).click();
+        cy.findByRole('alert').should('contain.text', 'Item successfully added');
+    });
+
+    it('can use all possibles configuration options', () => {
+        cy.findByRole('region', { 'name': "Request sources configuration" }).as("config");
+        cy.get('@config').getDropdownByLabelText('Request source').as("source_dropdown");
+
+        // Default value
+        cy.get('@source_dropdown').should(
+            'have.text',
+            'From template'
+        );
+
+        // Make sure hidden dropdowns are not displayed
+        cy.get('@config').getDropdownByLabelText('Select a request source...').should('not.exist');
+
+        // Switch to "Specific request source"
+        cy.get('@source_dropdown').selectDropdownValue('Specific request source');
+        cy.get('@config').getDropdownByLabelText('Select a request source...').as('specific_request_source_id_dropdown');
+        cy.get('@specific_request_source_id_dropdown').selectDropdownValue('E-Mail');
+
+        cy.findByRole('button', { 'name': 'Update item' }).click();
+        cy.get('@source_dropdown').should('have.text', 'Specific request source');
+        cy.get('@specific_request_source_id_dropdown').should('have.text', 'E-Mail');
+    });
+
+    it('can create ticket using default configuration', () => {
+        // Go to preview
+        cy.findByRole('tab', { 'name': "Form" }).click();
+        cy.findByRole('link', { 'name': "Preview" })
+            .invoke('removeAttr', 'target') // Cypress can't handle tab changes
+            .click();
+
+        cy.findByRole('button', { 'name': 'Send form' }).click();
+        cy.findByRole('link', { 'name': 'My test form' }).click();
+
+        // Check ticket values
+        cy.getDropdownByLabelText('Request source').should('have.text', 'Helpdesk');
```

Comment:
This test looks good but we are checking for the default "Helpdesk" value which will always be set for any created tickets that do not specify otherwise.

To be more a bit more resilient, maybe we could use `cy.createWithAPI` to create a predefined field on the default template (you can directly its id in the request, it wont change in the test database) with a different value ?